tests/unit/lms/services/blackboard_api/client_test.py

```diff
@@ -73,12 +70,13 @@ def test_it_with_pagination(self, svc, basic_client, blackboard_list_files_schem
             ),
             factories.requests.Response(json_data={}),
         ]
+        blackboard_files = [MagicMock() for _ in range(8)]
 
         # Each Blackboard API response contains a page of results.
         blackboard_list_files_schema.parse.side_effect = [
-            [sentinel.file_1, sentinel.file_2, sentinel.file_3],
-            [sentinel.file_4, sentinel.file_5, sentinel.file_6],
-            [sentinel.file_7, sentinel.file_8],
+            blackboard_files[0:3],
+            blackboard_files[3:6],
+            blackboard_files[6:8],
```

Comment:
Sentinels don't allow dict-like access but we still don't care about the values that much so MagicMock seems to do the trick

Reply:
```suggestion
            blackboard_files[:3],
            blackboard_files[3:6],
            blackboard_files[6:],
```
? Just so I don't have to guess about out by ones in my head